src/test/kotlin/org/opensearch/commons/alerting/action/GetWorkflowResponseTests.kt

```diff
@@ -20,6 +20,7 @@ class GetWorkflowResponseTests {
         val sin = StreamInput.wrap(out.bytes().toBytesRef().bytes)
         val newRes = GetWorkflowResponse(sin)
         Assertions.assertEquals("id", newRes.id)
+        Assertions.assertFalse(newRes.workflow!!.auditDelegateMonitorAlerts!!)
```

Comment:
We need another test where it checks if this is true when `auditDelegateMonitorAlerts` is not set.